src/tests/conformance/messages.rs

```diff
@@ -378,6 +378,116 @@ async fn correctly_lists_peers() {
     node.stop().await;
 }
 
+#[tokio::test]
+async fn get_blocks() {
+    // ZG-CONFORMANCE-015
+    //
+    // The node responds to `GetBlocks` requests with a list of blocks based on the provided range.
+    //
+    // We test the following conditions:
+    //  1. unlimited queries i.e. stop_hash = 0
+    //  2. range queries i.e. stop_hash = i
+    //  3. a forked chain (we submit a valid hash, followed by incorrect hashes)
+    //
+    // Test procedure:
+    //  1. Create a node and seed it with the testnet chain
+    //  2. Establish a peer node
+    //  3. For each test case:
+    //      a) send GetBlocks
+    //      b) receive Inv
+    //      c) assert Inv received matches expectations
+    //
+    // The test currently fails for both Zebra and zcashd.
+    //
+    // Current behaviour:
+    //
+    //  zcashd: Fails because it ignores requests when the last hash is the last hash it knows of. We expect it to return
```

Comment:
Wouldn't this be expected with `start < range < stop` as opposed to `<= stop`?

Reply:
I would still expect a reply; even if its just the empty set `Inv({})`. 

From the spec:
"If a block hash is found in the node's main chain, the list of its children is returned back via the inv message" - in this case the list is simply empty.